Feisty/GameDetails/GameDetailsViewController.swift

```diff
@@ -16,33 +16,34 @@ class GameDetailsViewController: UIViewController {
   internal var selectedGame: Game?
   
   private lazy var viewModel: GameDetailsViewModel = {
-    return GameDetailsViewModel(self, game: selectedGame)
+    
+    if let selectedGame = selectedGame {
+      return GameDetailsViewModel(self, selectedGame)
+    } else {
+      return GameDetailsViewModel(self, Game(appid: "N/A", name: "No Game"))
+    }
+    
   }()
 
   override func viewDidLoad() {
     super.viewDidLoad()
     
     viewModel.getGameData()
-
   }
 
   private func setGameDetails(name: String, appID: String) {
 
-    if let selectedGame = selectedGame {
-      gameNameLabel.text = selectedGame.name
-      lblAppID.text = selectedGame.appID
-    }
+    gameNameLabel.text = name
+    lblAppID.text = appID
 
   }
 
 }
 
 extension GameDetailsViewController: GameDetailsLoadedType {
   
-  func gameDetailsFound(gameDetails: (gameName: String, appID: String)) {
-    
-    setGameDetails(name: gameDetails.gameName, appID: gameDetails.appID)
-    
+  func gameDetailsFound(_ withGameName: String, _ andAppID: String) {
```

Comment:
If you plan on using `_` to ignore a param name then there is no need to have `withGameName` as it won't be used in the intended context.
Rather have `func gameDetailsFound(withGameName gameName: String, andAppID appID: String)` Or
`func gameDetailsFound(_ gameName: String, _ appID: String)`